src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -375,19 +430,22 @@ private LocalDate convertToDate(String string) throws DateTimeParseException {
      * @return an integer arraylist with valid indices
      * @throws NumberFormatException if non-integer value is present in the input
      */
-    public static ArrayList<Integer> checkIndices(String input, int max) throws NumberFormatException {
+    public static ArrayList<Integer> checkIndices(String input, int max) {
         ArrayList<Integer> rawIndices = new ArrayList<>();
         int index;
 
         String[] words = input.trim().split(WHITESPACE);
-
+        
         for (String word : words) {
-            // TODO - inform team using this
-            index = Integer.parseInt(word);
-            rawIndices.add(index);
+            try {
+                index = Integer.parseInt(word);
+                rawIndices.add(index);
+            } catch (NumberFormatException ignored) {
+                // Non-integer inputs are ignored and will not be added to the array list rawIndices.
```

Comment:
Could you print messages for NumberFormatException and ArrayIndexOutOfBounds? I was doing this just now so maybe you could just use these in the Messages class:

`public static final String MESSAGE_INVALID_INDICES = "Please enter numerical indices.";`
`public static final String MESSAGE_OUT_OF_BOUNDS_INDICES = "Please enter an index that is within range.";`